### ISSUE TITLE
what's output_multiplier_scale parameter for?

### ISSUE THREAD

**Issue by andy-yangz** [xai-org/grok-1#93]
confused about output_multiplier_scale, from the code they will multiply an output_multiplier_scale to logits before output.
and that output_multiplier=0.5773502691896257, same as apply a default temperature with sqrt(3)

but why? quite confused, anyone can help me to get clear about this
